pool/web/view/htmx/htmx.ml

```diff
@@ -127,6 +139,7 @@ let create
         ()
   in
   let default s = Option.value ~default:"" s in
+  let append_html = overridden_value in
```

Comment:
In case the contact didn't answer the question, show a hint to the administrator.

Reply:
Done